pkg/test/framework/components/mixer/mixer.go

```
@@ -138,11 +144,23 @@ func (c *kubeComponent) Init(ctx environment.ComponentContext, deps map[dependen
 
 	// TODO: Add support to connect to the Mixer istio-policy as well.
 	// See https://github.com/istio/istio/issues/6174
+	port := 0
+	svc, err := e.Accessor.GetService(e.IstioSystemNamespace, telemetryService)
+	if err == nil {
+		for _, portInfo := range svc.Spec.Ports {
+			if portInfo.Name == grpcPortName {
+				port = portInfo.TargetPort.IntValue()
+			}
+		}
+	} else {
```

This should fail when the service is not found.